offload/test/mapping/prelock.cpp

```diff
@@ -1,12 +1,9 @@
 // RUN: %libomptarget-compilexx-generic
 // RUN: %libomptarget-run-generic %fcheck-generic
 
-// UNSUPPORTED: aarch64-unknown-linux-gnu
-// UNSUPPORTED: aarch64-unknown-linux-gnu-LTO
+// REQURIES: gpu
 // UNSUPPORTED: nvptx64-nvidia-cuda
```

Comment:
Can we also use `nvidia-gpu` here?

Reply:
Yes, I that should work. I kept those long names because I wasn't sure which targets this test was meant to run on (seeing that host, `nvptx64-nvidia-cuda`, `nvptx64-nvidia-cuda-LTO`, and `amdgcn-amd-amdhsa`). I figured it was best to err on the side of caution.